docs/source/guide/prompts_draft.md

```diff
@@ -16,7 +16,9 @@ With your [Prompt created](prompts_create), you can begin drafting your prompt c
 ## Draft a prompt and generate predictions
 
 
-1. Select your base model. For a description of all OpenAI models, see [OpenAI's models overview](https://platform.openai.com/docs/models/models-overview).
+1. Select your base model. 
+
+    The models that appear depend on the [API keys](prompts_create#Model-provider-API-keys) that you have configured for your organization. For a description of all OpenAI models, see [OpenAI's models overview](https://platform.openai.com/docs/models/models-overview).
```

Comment:
Might be worth explaining here that there will be 1 Azure model listed per deployment you add